allocate.go

```diff
@@ -325,33 +326,44 @@ func ExecPath(path string) ExecAllocatorOption {
 }
 
 // findExecPath tries to find the Chrome browser somewhere in the current
-// system. It performs a rather agressive search, which is the same in all
-// systems. That may make it a bit slow, but it will only be run when creating a
-// new ExecAllocator.
+// system. Different searches are performed depending on OS runtime.
```

Comment:
`Different searches are performed depending on OS runtime.` -> `It finds in different locations on different OS systems.`